database/metadata/databases/default/tables/public_crashes_list_view.yaml

```diff
@@ -11,7 +11,6 @@ object_relationships:
         remote_table:
           name: recommendations
           schema: public
-array_relationships:
   - name: units
```

Comment:
this deletion is throwing me off. looks like this is being changed to an `object` relationship, and yet nothing appears to be broken. can you double-check this?

Reply:
thanks for catching this, definitely should be an array relationship. just fixed that and i also updated the relationship to use `crashes_list_view.id` and `units.crash_pk` instead of `cris_crash_id`.

Reply:
nice—i'm still confused by how this was not breaking things, but it all looks great. i re-tested the crash list export to double-check there and everything looks good. thank you!